### ISSUE TITLE
[EN][Fishing] Loot

### ISSUE THREAD

**Issue by xolon** [Atlantiss/BugTracker#7005]
Sometimes loot window appears but after 1 second it closes and it says "No fish are hooked"
http://imgur.com/a/3cYl4

**Comment by 1Andas** [comment]
Yeah confirmed.

**Comment by jissabraham** [comment]
yes it happens i can see the loot. and also in TB some times corpse also fishes up
